en/html/README.md

```diff
@@ -167,10 +167,10 @@ Make sure you're in the `djangogirls` directory and let's tell `git` to include
 
 {% filename %}command-line{% endfilename %}
 ```
-$ git add --all .
+$ git add .
 ```
 
-> __Note__ `--all` means that `git` will also recognize if you've deleted files (by default, it only recognizes new/modified files). Also remember (from chapter 3) that `.` means the current directory.
+> __Note__ By default, `git add` will recognize if you've deleted files (by default, it only recognizes new/modified files). Also remember (from chapter 3) that `.` means the current directory.
```

Comment:
This is contradictory. Should probably be
```suggestion
> __Note__ By default, `git add` will recognize if you've deleted files. Also remember (from chapter 3) that `.` means the current directory.
```

Reply:
Your suggestion is accurate.